### PR TITLE
{bio}[foss/2018b] SMARTdenovo v20180219

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 9.7, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/375c00e85c6837af40767d82fc94f99a for a full test report.

**Comment by vanzod** [comment]
@akesandgren It looks like the code changes over time...

**Comment by akesandgren** [comment]
Crap, I forgot that inject-checksum moves my comments around.
The checksum will NOT match since it generates the tar file on the fly every time you download it.
There is a comment about this, but it got moved to the wrong place.

**Comment by akesandgren** [comment]
So, I'm bnot sure how we should handle this one. --from-pr -u won't work since the checksum is always wrong, and tests will complain if there are no checksums...

**Comment by akesandgren** [comment]
And they refused to produce a release... I already tried.

File: easybuild/easyconfigs/s/SMARTdenovo/SMARTdenovo-20180219-foss-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+tools to generate accurate consensus sequences, though a platform
+dependent consensus polish tools (e.g. Quiver for PacBio or Nanopolish
+for ONT) are still required for higher accuracy."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+commit = '5cc1356bffb87197b0fb9b835cfe1adc5a764dc6'
+sources = [{
+    'filename': '%%(name)s-master-20190215-%s.tar.gz' % commit,
+    'git_config': {
```
Comment (boegel) [inline easybuild/easyconfigs/s/SMARTdenovo/SMARTdenovo-20180219-foss-2018b.eb]: @akesandgren When the sources live on GitHub you don't need to use `git` at all, you can easily get a tarball for a specific commit (with a fixed checksum)

Done in https://github.com/akesandgren/easybuild-easyconfigs/pull/9

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3106.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/262490865d43102e8184f9f99151e9a4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2609.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/f0f32059a4fdb568f9aa78435ec77f6a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2008.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/1103bf078a4c83a4f5c94b25e26fc164 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**